justfile

```diff
@@ -23,12 +27,12 @@ gen: _protoc _protoc_gen_go _protoc_gen_go_grpc && lint
 
 test: gen
   go test -v ./...
-  cargo test -- --nocapture
+  cargo test --features bolero -- --nocapture
```

Comment:
This is my main concern with the PR. I don't like this, for the reasons we discussed in the dataplane repo: this means that every time we run tests, including in CI, we exercise the fuzzing tests (or did I misunderstand?). This is not necessarily what I want when testing locally; nor is this what I expect (or desire) from CI runs for Pull Requests.

Have you considered making it a separate target, running in a separate GitHub workflow?